### ISSUE TITLE
Python 2.7: ccxt.[exchange]() call error (ver 1.9.163)

### ISSUE THREAD

**Issue by PhinixPhire** [ccxt-dev/ccxt#331]
I just updated to the latest ccxt (1.9.163) and now I'm getting errors such as this whenever I try to initialize an exchange:

Code: 
```
import ccxt
southxchange = ccxt.southxchange()

```
Result:
```
Traceback (most recent call last):
  File "C:/Users/Phinix/Desktop/Crypto/CCXTtests.py", line 3, in <module>
    southxchange = ccxt.southxchange()
  File "C:\Python27\lib\site-packages\ccxt\exchanges.py", line 17538, in __init__
    super(southxchange, self).__init__(params)
  File "C:\Python27\lib\site-packages\ccxt\exchange.py", line 163, in __init__
    for attr in self.__dir__():
AttributeError: 'southxchange' object has no attribute '__dir__'
>>> 
```

**Comment by kroitor** [comment]
This is a bug due to one of recent commits, it will be fixed right away! Thx for reporting!

**Comment by PhinixPhire** [comment]
Thanks, Igor! 
I'll be sure to include the Python version in my Issue titles moving forward for you too. :)

**Comment by kroitor** [comment]
@PhinixPhire I can see it from the output, no worries ) Fixing it.

**Comment by kroitor** [comment]
I've uploaded the fix for it, but GitHub and Travis now say:

<img width="576" alt="screen shot 2017-10-17 at 19 32 04" src="https://user-images.githubusercontent.com/1294454/31676958-ed2792ae-b371-11e7-826d-9db522ca76f9.png">

Have to wait a little (

**Comment by kroitor** [comment]
Ok, this is fixed in version 1.9.165+. Sorry for the inconvenience. It was a leftover minor bug. Thanks for your patience! Let us know if you have any other questions or difficulties. Best wishes!

**Comment by PhinixPhire** [comment]
No problem at all. I appreciate the quick response, sir!
